posts/2021-04-19-JAXRS-multipart-payloads-21005-beta.adoc

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta - OpenLiberty.io
+seo-description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 Support for `oauth-2.0` and `samlWeb-2.0`.  
+blog_description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 Support for `oauth-2.0` and `samlWeb-2.0`.  
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 Support for `oauth-2.0` and `samlWeb-2.0`.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes 
+
+* <<jee9support, Jakarta EE 9 Support for oauth-2.0 and samlWeb-2.0>>
+* <<webservices, Build multipart payloads for your JAX-RS client and services>>.
+
+
+[#webservices]
+=== Build multipart payloads for your JAX-RS client and services
+
+Often, a RESTful service or client will need to send multiple disparate pieces of data in the same request, for example uploading a resume/CV with a picture, and text for name and address. This is usually done using `multipart/form-data`.
```

Comment:
request, for example, uploading a resume/CV with a picture and text for name and address.

Reply:
Done